docs/plugins/language.md

```diff
@@ -1,49 +1,48 @@
 # Language
 
-The Language plugin supports internationalisation features for your App.
+The Language plugin supports internationalization features for your App.
 
 ## Usage
 
-In order to use the Language plugin, import it from the Lightning SDK.
+If you want to use the Language plugin, import it from the Lightning SDK:
 
-```js
+```
 import { Language } from '@lightningjs/sdk'
 ```
 
-### Loading on boot
+### Loading on Boot
 
 You can automatically load and initialize the Language plugin when your App boots, by specifying a `static` method `language()`
-on the `App`-class in `src/App.js`.
+on the `App` class in **src/App.js**.
 
-The `language`-method should return which language to use on initialisation. This can either be a string or a Promise that resolves a
+The `language` method returns the language to be used on initialization. This can either be a string or a promise that resolves a
 string with the language code.
 
-
-```js
-export default class App extends Lightning.Component {
+```
+export default class App extends Lightning.component {
 
   static language() {
     // hardcoded string
     return 'en'
-    // or string retrieved van Storage
-    return Storage.get('selectedLanguage') || 'en'
+    // or string retrieved from Storage
+    return Storage.get('selectedLanguage')  || 'en'
     // or a promise (e.g. using the Profile plugin)
     return Profile.language()
   }
 
 }
 ```
 
-### Translations file
+### Translations File
 
-By default the Language plugin expects a `json` file, called `translations.json` in the `static` folder of your App.
+By default, the Language plugin expects a JSON translations file, called **translations.json**, which is located in the **static** folder of your App.
```

Comment:
suggestion: expects a file with translations in JSON format, called ...

Reply:
Corrected accordingly.